x-pack/plugins/apm/public/components/fleet_integration/apm_agents/index.tsx

```diff
@@ -0,0 +1,326 @@
+  elasticWebsiteUrl,
+  variantId,
+}: {
+  newPolicy: NewPackagePolicy;
+  createAgentInstructions: CreateAgentInstructions;
+  elasticWebsiteUrl: string;
+  variantId: string;
+}) {
+  const vars = newPolicy?.inputs?.[0]?.vars;
+  const apmServerUrl = vars?.url.value;
+  const secretToken = vars?.secret_token.value;
+  const steps = createAgentInstructions(apmServerUrl, secretToken);
+
+  return (
+    <>
+      <EuiSpacer />
+      {steps.map(
+        ({ title, textPre, textPost, customComponentName, commands }) => {
+          const commandBlock = Array.isArray(commands)
+            ? commands.join('\n')
+            : commands;
+          return (
+            <section>
+              <EuiText>
+                <h4>{title}</h4>
+              </EuiText>
+              <EuiSpacer size="s" />
+              <EuiText color="subdued" size="s">
+                {textPre && (
+                  <EuiFlexGroup justifyContent="spaceBetween">
+                    <EuiFlexItem grow={false}>
+                      <Content
+                        text={replaceTemplateStrings(
+                          textPre,
+                          elasticWebsiteUrl
+                        )}
+                      />
+                    </EuiFlexItem>
+                    {commandBlock && (
+                      <EuiFlexItem grow={false}>
+                        <CopyCommands commands={commandBlock} />
+                      </EuiFlexItem>
+                    )}
+                  </EuiFlexGroup>
+                )}
+                {commandBlock && (
+                  <>
+                    <EuiSpacer size="s" />
+                    <EuiCodeBlock language="bash" data-test-subj="commands">
+                      {commandBlock}
+                    </EuiCodeBlock>
+                  </>
+                )}
+                {customComponentName === 'TutorialConfigAgent' && (
+                  <TutorialConfigAgent
+                    variantId={variantId}
+                    apmServerUrl={apmServerUrl}
+                    secretToken={secretToken}
+                  />
+                )}
+                {customComponentName === 'TutorialConfigAgentRumScript' && (
+                  <TutorialConfigAgent
+                    variantId="js_script"
+                    apmServerUrl={apmServerUrl}
+                    secretToken={secretToken}
+                  />
+                )}
+                {textPost && (
+                  <>
+                    <EuiSpacer />
+                    <Content
+                      text={replaceTemplateStrings(textPost, elasticWebsiteUrl)}
+                    />
+                  </>
+                )}
+              </EuiText>
+              <EuiSpacer />
+            </section>
+          );
+        }
+      )}
+    </>
+  );
+}
+
+function replaceTemplateStrings(text: string, elasticWebsiteUrl: string) {
+  return text.replaceAll('{config.docs.base_url}', elasticWebsiteUrl);
+}
```

Comment:
This is a stub for `replaceTemplateStrings` at https://github.com/elastic/kibana/blob/master/src/plugins/home/public/application/components/tutorial/replace_template_strings.js#L24, which needs work to be accessible.